### PR TITLE
Fix `scripts/get-default-key-address.sh` & `scripts/genesis.sh`

### PR DESCRIPTION
**Pull request by Jimbo4350**
#411  & #412

### CODE REVIEW

**Comment by ArturWieczorek** [comment]
`genesis.sh` now works fine but the other one  still got issues:

> ./scripts/get-default-key-address.sh configuration/genesis/delegate-keys.001.key 
> cat: /home/artur/Projects/Fix: No such file or directory
> cat: Script/cardano-node/configuration/genesis/GENHASH: No such file or directory
> 2cWKMJemoBahs2gd4sQX1XgSLVLyAMrtNy2qgp4t2BYjBFygD3dLo4ejXC4p3C3pnc6qd

I managed to fix it locally by modifying lib-node.sh: 

`genesis_hash="cat ${genesis_root}/GENHASH"` - surrounded it with double quotes instead of backticks

File: scripts/lib-node.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ then if test -x ./scripts/genesis.sh
      fi
 fi
 
-genesis_hash=`cat ${genesis_root}/GENHASH`
+genesis_hash="cat ${genesis_root}/GENHASH"
```
Comment (CodiePP) [inline scripts/lib-node.sh]: that will not work! try it in a shell and you will see that the $genesis_hash will contain a string: "cat configuration/Something/GENHASH", but not the genesis' hash.

Comment (CodiePP) [reply scripts/lib-node.sh]: the meaning of backticks is: run the command and get its output
whereas double quotes are only to enclose strings

Comment (ArturWieczorek) [reply scripts/lib-node.sh]: @CodiePP Hi, after branch checkout I ran those scripts again (`genesis.sh` and then `get-default-key-address.sh`)  and everything works fine... **HOWEVER** you are right - this line is still wrong. Thanks for pointing this.

Leaving it like it was before change: `` `cat ${genesis_root}/GENHASH` `` will not work for directory with white spaces in name but I think the change below should work for all cases:

`` genesis_hash=`cat "${genesis_root}/GENHASH"` ``

Comment (CodiePP) [reply scripts/lib-node.sh]: 👍

**Pull request merged**